ibazel/main.go

```diff
@@ -59,6 +59,25 @@ ibazel build //path/to/my/buildable:target
 `)
 }
 
+type SourceEventProcessor struct {
+	SourceFileEvents chan fsnotify.Event
+	SourceFileWatcher *fsnotify.Watcher
+}
+
+func (s SourceEventProcessor) Listen() {
```

Comment:
Can you add a method that creates a SourceEventProcessor and creates the channel then calls

go s.Listen()